pha-admin/src/main/java/com/store/pharmacy/Company/serviceImpl/CompanyServiceImpl.java

```diff
@@ -0,0 +1,91 @@
+package com.store.pharmacy.Company.serviceImpl;
+
+import com.store.pharmacy.Company.model.Company;
+import com.store.pharmacy.Company.model.CompanyInput;
+import com.store.pharmacy.Company.model.CompanyOutput;
+import com.store.pharmacy.Company.repository.CompanyRepository;
+import com.store.pharmacy.Company.service.CompanyService;
+import com.store.pharmacy.exception.DataNotFoundException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.LocalDateTime;
+
+@Service
+@Transactional
+public class CompanyServiceImpl implements CompanyService {
+    @Autowired
+    CompanyRepository companyRepository;
+
+    private final String INSERT = "INSERT";
+    private final String UPDATE = "UPDATE";
+
+    @Override
+    public CompanyOutput save(CompanyInput companyInput) {
+        Company inParam = convertCompanyInputToCompany(null, companyInput, INSERT);
```

Comment:
"NullPointerException" will be thrown when invoking method convertCompanyInputToCompany.

<img width="668" alt="image" src="https://user-images.githubusercontent.com/86360944/137624355-f3965de5-c9e2-4796-999c-786266c80784.png">
<img width="776" alt="image" src="https://user-images.githubusercontent.com/86360944/137624393-62cf9583-3040-41fc-949e-2e8a19424ebe.png">